manifest.json

```diff
@@ -325,6 +325,16 @@
         "baseImage": "public.ecr.aws/lambda/provided:al2023"
       },
       "hasSpecificImageBuild": true
+    },
+    {
+      "displayName": "graalvm java21 (prov.al2)",
+      "runtime": "provided.al2",
```

Comment:
Since there is a new `provided.al2023` runtime, let's use this one instead of `provided.al2`?